Tella/Data/Networking/APICall/APIError.swift

```diff
@@ -21,16 +24,23 @@ extension APIError: LocalizedError {
             return customErrorMessage(errorCode: code)
         case .unexpectedResponse:
             return "Unexpected response from the server"
+        case .error(let message):
+            return message
         case .noInternetConnection:
             return LocalizableSettings.settServerNoInternetConnection.localized
+        case .badServer:
+            return LocalizableSettings.settServerServerURLIncorrect.localized
         }
     }
-    func customErrorMessage(errorCode : Int) -> String {
-        switch HTTPErrorCodes(rawValue: errorCode) {
+    private func customErrorMessage(errorCode : Int) -> String {
+        let httpErrorCode = HTTPErrorCodes(rawValue: errorCode)
+        switch httpErrorCode{
         case .unauthorized:
             return "Invalid username or password"
         case .forbidden:
             return "Account locked due to too many unsuccessful attempts."
+        case .notFound:
+            return LocalizableSettings.settServerServerURLIncorrect.localized
         default:
             return "Custom Error"
```

Comment:
What do you think if we change "Custom Error" with "Unexpected response from the server"